src/api/llm/http.ts

```diff
@@ -72,3 +72,28 @@ export async function sendUserRating(
     throw new Error('Unexpected status code: ' + resp.status);
   }
 }
+
+export async function fetchAppNames(creds: Credentials) {
```

Comment:
1. Imho a misnomer, since you return more than just the names. Whenever I look at this, I expect a list of strings to be returned. Would recommend a different name 🙏 
2. Won't this return both VDC and RDC type apps? Are we fully supporting VDC now?

Reply:
@alexplischke are there apps which are RDC or VDC specific? Do you have an example app that is VDC only? I'd be happy to test uploading and filtering that out.

Reply:
Ah, zips are simulators only. I'll filter those out for now, with a comment to remove for when VDC is enabled.